templates/html/templates/common/instance_view_builder.html

```diff
@@ -376,18 +378,19 @@
     }" %}
     {% set content, error = from_endpoint(conf.default_endpoint).with_query(select_query, parameters).fetch_tabular() %}
     {% call macros.render_fetch_results(content, error) %}
-        {% if not content.dropna(how="all").empty %}
+        {% if not content.dropna().empty %}
             <div class="card">
                 <div class="card-header">
                     <i class="fa fa-caret-down" aria-hidden="true"></i>
                     <a class="text-dark font-weight-bold" onclick="showContent()">
-                        Useful links: Classified With | Property Configuration
+                        Useful links
                     </a>
                 </div>
                 <div class="collapse">
                     <div class="card-body">
                         {% for row in content.itertuples() %}
                             <a href="#{{ row.link }}">{{ row.linkLabel }}</a><br>
+                            {{ "<hr> " if not loop.last }}
```

Comment:
Well done! You have discovered the special loop variable inside for loops. It is quite handy, many nifty things can be done with it. GJ using it ;)